scripts/package-providers.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
```

Comment:
not really going to pick this one apart since you said you want to do a separate refactor of it